resources/views/Staff/report/show.blade.php

```diff
@@ -98,27 +98,99 @@ class="form"
+                <dd>{{ $report->id }}</dd>
+            </div>
+            <div class="key-value__group">
+                <dt>{{ __('ticket.category') }}</dt>
+                <dd>{{ $report->type }}</dd>
+            </div>
+            <div class="key-value__group">
+                <dt>{{ __('common.created_at') }}</dt>
+                <dd>{{ $report->created_at->format('Y-m-d') }}</dd>
+            </div>
+            <div class="key-value__group">
+                <dt>{{ __('common.reporter') }}</dt>
+                <dd>
+                    <x-user-tag :anon="false" :user="$report->reporter" />
+                </dd>
+            </div>
+            <div class="key-value__group">
+                <dt>Reported</dt>
+                <dd>
+                    <x-user-tag :anon="false" :user="$report->reported" />
+                </dd>
+            </div>
+            @if ($report->closed_by !== null)
+                <div class="key-value__group">
+                    <dt>Closed by</dt>
+                    <dd>
+                        <x-user-tag :anon="false" :user="$report->judge" />
+                    </dd>
+                </div>
+                <div class="key-value__group">
+                    <dt>{{ __('ticket.closed') }}</dt>
+                    <dd>
+                        <time
+                            datetime="{{ $report->closed_at }}"
+                            title="{{ $report->closed_at }}"
+                        >
+                            {{ $report->closed_at?->format('Y-m-d') }}
+                        </time>
+                    </dd>
+                </div>
+            @endif
+        </dl>
     </section>
     <section class="panelV2">
-        <h2 class="panel__heading">Solved by</h2>
+        <h2 class="panel__heading">{{ __('common.actions') }}</h2>
         <div class="panel__body">
-            @if ($report->solved)
-                <x-user-tag :anon="false" :user="$report->staff" />
-            @else
-                <span class="text-red">
-                    <i class="{{ config('other.font-awesome') }} fa-times"></i>
-                    UNSOLVED
-                </span>
+            <form
+                class="form form--horizontal"
+                action="{{ route('staff.reports.assignee.store', ['report' => $report]) }}"
+                method="POST"
+                x-data
+            >
+                @csrf
+                <p class="form__group">
+                    <select
+                        id="staff_id"
+                        name="staff_id"
+                        class="form__select"
+                        x-on:change="$root.submit()"
+                    >
+                        <option hidden disabled selected value=""></option>
+                        @foreach (App\Models\User::select(['id', 'username'])->whereIn('group_id', App\Models\Group::where('is_modo', 1)->whereNotIn('id', [9])->pluck('id')->toArray())->get() as $user)
```

Comment:
Can this query be moved into the controller instead? Also, the group slug should be used instead of the id.

Reply:
Done.